golang/cmd/factoryinsight/http.go

```diff
@@ -2044,12 +2046,44 @@ func processUniqueProductsWithTagsRequest(c *gin.Context, getDataRequest getData
 		return
 	}
 
-
 	// Fetching from the database
 	uniqueProductsWithTags, err := GetUniqueProductsWithTags(span, getDataRequest.Customer, getDataRequest.Location, getDataRequest.Asset, getUniqueProductsWithTagsRequest.From, getUniqueProductsWithTagsRequest.To)
 	if err != nil {
 		handleInternalServerError(span, c, err)
 		return
 	}
 	c.JSON(http.StatusOK, uniqueProductsWithTags)
-}
\ No newline at end of file
+}
+
+type getProcessAccumulatedProducts struct {
+	From time.Time `form:"from" binding:"required"`
+	To   time.Time `form:"to" binding:"required"`
+}
+
+func processAccumulatedProducts(c *gin.Context, getDataRequest getDataRequest) {
+	// Jaeger tracing
+	var span opentracing.Span
+	if cspan, ok := c.Get("tracing-context"); ok {
+		span = ginopentracing.StartSpanWithParent(cspan.(opentracing.Span).Context(), "processUniqueProductsWithTagsRequest", c.Request.Method, c.Request.URL.Path)
+	} else {
+		span = ginopentracing.StartSpanWithHeader(&c.Request.Header, "processUniqueProductsWithTagsRequest", c.Request.Method, c.Request.URL.Path)
+	}
+	defer span.Finish()
+
+	var getProcessAccumulatedProducts getProcessAccumulatedProducts
+	var err error
+
+	err = c.BindQuery(&getProcessAccumulatedProducts)
+	if err != nil {
+		handleInvalidInputError(span, c, err)
+		return
+	}
+
+	accumulatedProducts, err := GetAccumulatedProducts(span, getDataRequest.Customer, getDataRequest.Location, getDataRequest.Asset, getProcessAccumulatedProducts.From, getProcessAccumulatedProducts.To)
```

Comment:
Where is this function?

Reply:
Inside database.go.
Every other http handle also does not specify this.